source/Sashimi.Tests/ActionHandlersFixture.cs

```diff
@@ -380,15 +450,15 @@ void PopulateVariables(TestActionHandlerContext<Program> _)
                 _.Variables.Add("Octopus.Action.Aws.Region", "ap-southeast-1");
                 _.Variables.Add("Hello", "Hello World from AWS");
                 _.Variables.Add("bucket_name", bucketName);
-                _.Variables.Add(TerraformSpecialVariables.Action.Terraform.VarFiles, "example.tfvars");
+                _.Variables.Add(TerraformSpecialVariables.Action.Terraform.VarFiles, "example.tfvars"); // TODO: replace file with the original after substitution
                 _.Variables.Add(TerraformSpecialVariables.Action.Terraform.AWSManagedAccount, "AWS");
                 _.Variables.Add(KnownVariables.OriginalPackageDirectoryPath, temporaryFolder.DirectoryPath);
             }
 
-            var output = ExecuteAndReturnResult(typeof(TerraformPlanActionHandler), PopulateVariables, "AWS");
+            var output = ExecuteAndReturnResult(typeof(TerraformPlanActionHandler), PopulateVariables, temporaryFolder.DirectoryPath);
```

Comment:
I don't really follow why we changed this?

Reply:
The test was doing variables substitution directly on the folder, so any subsequent test runs failed as the files were already modified. I think the intention here already was to copy to a temp folder, so I just changed parameters to make it so.